packages/devextreme/js/__internal/ui/chat/chat_message_group.ts

```diff
@@ -118,11 +123,29 @@ class MessageGroup extends Widget<MessageGroupOptions> {
     $messageGroupInformation.appendTo(this.element());
   }
 
+  _removePrevBubbleLastClass(): void {
+    const $bubbles = $(this.element()).find(`.${CHAT_MESSAGE_BUBBLE_CLASS}`);
+    const $lastBubble = $bubbles.eq($bubbles.length - 1);
+
+    $lastBubble.removeClass(CHAT_MESSAGE_BUBBLE_LAST_CLASS);
```

Comment:
Can't we handle this with CSS for applying borders correctly? It seems we can. What is the reason for using classes here?

Reply:
Okay, we'll assume that you know what you're doing